src/first_steps/expressions.md

```diff
@@ -32,8 +32,8 @@ Supported arithmetic operations are:
  *  \* : multiplication
  *  / : division
  *  % : modulus
- *  \> : shift right
- *  < : shift left
+ *  \>> : shift right
+ *  << : shift left
```

Comment:
there's also <<< and >>> which rotate the bits instead of shifting them. and im now working on fixing < and > because that may result on 1 or 0 as its a comparison operator (and adding tests)

Reply:
mind to update the doc with those operators too?

Reply:
no need to wait that much to merge this, ill update the esil documentation when i get some spare time. thanks for the contribution!

Reply:
@trufae somehow missed your comment about `>>>`
Let me check them and add into doc as well

Reply:
Cool